crates/ironrdp-server/src/encoder/mod.rs

```diff
@@ -114,7 +114,7 @@ impl UpdateEncoder {
     }
 
     pub(crate) fn bitmap(&mut self, bitmap: BitmapUpdate) -> Result<UpdateFragmenter<'_>> {
-        self.bitmap_updater.handle(bitmap, &mut self.pdu_encoder)
+        self.bitmap_updater.handle(&bitmap, &mut self.pdu_encoder)
     }
```

Comment:
question: Any reason to not take BitmapUpdate by reference here too?

Reply:
There is more refactoring on this area with the following patches, but I can include that here too I guess.

Reply:
actually, the patch right after requires to pass the update.